docs/README-EDITING.md

```diff
@@ -0,0 +1,174 @@
+
+To display a published documentation version in a browser:
+1. Start a [local Handsontable documentation server](./README.md#getting-started-with-handsontable-docs).
+2. In your browser, go to http://localhost:8080/docs/{semver.version}/.
+
+If you're editing the `latest` version (a version with the largest `<semver.version>` number), remember to make the same edits to the `next` version as well.
+
+## Editing the API reference
+
+The API reference is generated automatically from the source code, into the `/next/api/` directory.
+
+To edit the API reference, go into the source code and change the JSDoc comments.
+
+To generate the API reference output, run the following command from the `/docs/` directory:
+
+```bash
+npm run docs:api
+```
+
+### JSDoc links
+
+As we want to support case-insensitive URLs, follow these guidelines when editing the JSDoc links:
+
+* Use lower case characters.
+* To separate words before `#`, use `-`.
+
+```js
+// Wrong:
+/** {@link Options#autoColumnSize}. */
+
+// OK: 
+/** {@link options#autocolumnsize Options#autoColumnSize}. */
+
+
+// Wrong:
+/** {@link HiddenRows} */
+
+// OK:
+/** {@link hidden-rows HiddenRows} */
+```
+
+## Documentation versioning
+
+To create a new version of the Handsontable documentation:
+
+* From the `handsontable/docs` directory, run:
+```bash
+npm run docs:version <semver.version>
+# for example:
+# npm run docs:version 9.0
+```
+
+To remove an existing version of the Handsontable documentation:
+
+* Remove the required version's [directory](./README.md#handsontable-docs-directory-structure):
+```bash
+rm -rf ./<semver.version>
+```
+
+## Markdown containers
+
+To render content in different ways, the documentation uses custom Markdown containers, for example:
+
+```markdown
+::: example #exampleId .class :preset --html 1 --js 2
+
+// code example
+
+:::
+```
+
+We use the following Markdown containers:
+
+| Container                    | Usage                                                      |
+|------------------------------|------------------------------------------------------------|
+| `::: tip`                    | A note.                                                    |
+| `::: example [options]`      | Renders a code example as specified in [options].          |
+| `::: source-code-link [URL]` | Adds a source code link to the right of an API ref header. |
+
+### Adding code examples
+Using the `example` Markdown container, you can add code snippets that show the code's result:
+
+```js
+::: example #exampleId .class :preset --html 1 --js 2 --css 3  --no-edit --tab preview
+// `#example1` is the example's ID, for creating a Handsontable container
+    ```html
+    <div id="example1"></div>
+    ```
+    ```js
+    // code here
+    ```
+    ```css
+    /* custom css here */
+    ```
+:::
+```
+
+The `example` Markdown container offers the following options:
```

Comment:
Referring to possible options to set as a link to the JS file where you have to analyze the file to find these options is not a good practice, in my opinion.

Reply:
@swistach - sure, agreed: https://github.com/handsontable/handsontable/pull/8350